lib/components/SelectionControl/checkbox/checkbox-single/index.tsx

```diff
@@ -35,6 +35,7 @@ export interface CheckboxProps
             | React.KeyboardEvent<HTMLSpanElement>,
     ) => void;
     className?: string;
+    checkboxPosition?: "left" | "right";
```

Comment:
@kate-deriv  we have recently added, position types in `@types` can we use that?

Reply:
Hi! Sure, sorry, I didn't know about it. Will add)

Reply:
@prince-deriv done 🙇🏻‍♀️